tests/lcs/core/field/htmlText.livecodescript

```diff
@@ -55,6 +55,31 @@ on TestSurrogateRoundtrip
    delete stack "Test"
 end TestSurrogateRoundtrip
 
+on TestListStyleRoundtrip
+   create stack "Test"
+   set the defaultStack to "Test"
+   
+   create field "TestField"
+   
+   set the text of field "TestField" to "one" & CR & "two" & CR & "three"
+   set the listStyle of line 1 of field "TestField" to "skip"
+   set the listStyle of line 2 of field "TestField" to "disc"
+   TestAssert "htmlText roundtrips listStyle beginning with skip", the htmlText of field "TestField" is the htmlText of field  "TestField"
```

Comment:
@livecodepanos I'm not sure I get this test. Isn't the htmlText of the field always going to equal the htmlText of the field?

Reply:
Lol, you are right, this was definitely not what I meant to write!